css/style.css

```diff
@@ -16,4 +17,187 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* header */
+
+ header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 2rem 5rem;
+ }
+
+ .logoweb{
+    display: block;
+    min-width: 2rem;
+    min-height: 2rem;
+ }
+
+ /* navigation */
+
+ .nav {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    list-style: none;
+
+ }
+
+ .nav-item {
+    display: flex;
+    padding: 0 1.5rem;
+    
+ }
+
+ .nav-link {
+    color: var(--grey);
+    text-decoration: none;
+ }
+
+ .nav-link:hover {
+    color: var(--orange);
+ }
+
+ .nav-item:first-child > a {
+    color: var(--grey-dark);
+ }
+
+ .nav-item:first-child > a:hover{
+    color: var(--orange);
+ }
+
+ /* main */
+
+ main {
+    display: flex;
+    flex-direction: column;
+    
+ }
+
+ /* section */
+
+.hero {
+    background-image: url(/img/first-background.jpg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    padding: 12rem 0;
+    color: var(--white);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 2rem;
+    margin: 0;
+}
+
+.hero h2 {
+    font-weight: 200;
+    font-size: 35px;
+    text-align: center;
+}
+
+.hero h5 {
+    margin-top: -1rem;
+    margin-bottom: 2rem;
+    font-weight: 50;
+    font-size: 20px;
+    
+}
+
+.hero button {
+    background-color: var(--orange);
+    border: none;
+    padding: .8rem 1rem;
+    border-radius: 5px;
+}
+.hero button:hover {
+    background-color: var(--grey-dark);
+}
+
+.hero button a {
+    text-decoration: none;
+    color: var(--white);
+}
+
+section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 4rem;
+    margin-bottom: 3rem;
+}
+
+section h2 {
+    font-weight: 200;
+    font-size: 35px;
+}
+
+article {
+    display: flex;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    
+    
+}
+
+article h5 {
+    font-weight: 200;
+    font-size: 20px;
+    
+}
+
+article img {
+    width: 120px;
+    height: 120px;
+}
+
+.context {
+    margin: 3rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 1.5rem;
```

Comment:
https://www.loom.com/share/ee2acf6c18864fe78ae206f5d6fba9df
I would use text-align: center and give each of these boxes the same size.
I.E.

width: 200px
text-align: center

so that the text underneath is centered with this